src/main/openapi/openapi_v1.yaml

```diff
@@ -148,9 +160,9 @@ paths:
           required: false
           schema:
             type: string
-            example: '*, [child]'
-            default: '*, [child]'
-
+            example: 'id,title'
```

Comment:
Same problem as for `start`: Only the fields `id,title` will be used per default. One workaround is to provide multiple example values, the first one being `*`. That way the user can see what is possible _and_ the GUI-default will be usable.

Reply:
fixed.

Reply:
The Open*API documentation explains how to add multiple examples: https://swagger.io/docs/specification/adding-examples/ and yes, `examples` (plural) works. The confusing part is the indentation. I have added multiple examples for `q` to the branch - take a look at the GUI, it's nicely presented and I think we should use it at least for `fl`.

If you add it for `fl`, remember to use arrays `['year:1990']` as values. The empty array `[]` works too and should probably be the first example.